Emby.Server.Implementations/HttpServer/Security/AuthorizationContext.cs

```diff
@@ -176,7 +210,32 @@ private AuthorizationInfo GetAuthorization(IRequest httpReq)
         /// <returns>Dictionary{System.StringSystem.String}.</returns>
         private Dictionary<string, string> GetAuthorizationDictionary(IRequest httpReq)
         {
-            var auth = httpReq.Headers["X-Emby-Authorization"];
+            var auth = httpReq.Headers["X-Jellyfin-Authorization"];
```

Comment:
What is the motivation for this?

Reply:
Does it break existing clients? I have no clue.

Reply:
It shouldn't break existing, as below if the auth is not set it tries to get **X-Emby-Authorization**

I can see the desire to remove references to Emby where possible and switch to Jellyfin

Reply:
All this does is allow the client to send Jellyfin, all previous auth headers are still supported

Reply:
A more modern authentication system would probably be based on JWT though. I don't see the benefit in renaming the old header.

Reply:
Since this is just a temporary API move I see what you mean, I will revert the auth header addition